CMakeLists.txt

```diff
@@ -158,18 +188,26 @@ if(CUDA_FOUND)
     include_directories(${NvToolExt_INCLUDE_DIR})
     link_libraries(${NvToolExt_LIBRARIES})
 
-    get_third_party(cub)
-    set(CUB_ROOT_DIR "${CMAKE_CURRENT_BINARY_DIR}/cub")
+    if (NOT CONDA_ROOT)
+        get_third_party(cub)
+        set(CUB_ROOT_DIR "${CMAKE_CURRENT_BINARY_DIR}/cub")
+    endif()
     find_package(CUB REQUIRED)
     include_directories(${CUB_INCLUDE_DIR})
 endif()
 
 add_definitions(-DKALDI_NO_PORTAUDIO=1)
 
-include(VersionHelper)
-get_version() # this will set KALDI_VERSION and KALDI_PATCH_NUMBER
-if(${KALDI_USE_PATCH_NUMBER})
-    set(KALDI_VERSION "${KALDI_VERSION}.${KALDI_PATCH_NUMBER}")
+if(OVERRIDE_KALDI_VERSION)
```

Comment:
Consider using just `KALDI_VERSION`? Is the additional variable really needed?

Reply:
This was added by @mmcauliffe, but I think the issue was (IIRC) that `KALDI_VERSION` doesn't get populated if there's no `git` installed.

Reply:
Yeah I think @kkm000 's point was that we could just set `KALDI_VERSION` via CMake `-DKALDI_VERSION` instead of introducing another variable, which was just an oversight on my part as I was learning how cmake works.  I'll fix that up.